### ISSUE TITLE
How to escape backticks in code block ?

### ISSUE THREAD

**Issue by Outlivier** [jonschlinkert/remarkable#146]
Hi, 
I would like to document markdown in markdown, how to write this with remarkable (i also don't now how to do it with github, so how to obtain this output without the backslashes) ?
 
``` markdown
# markdown not parsed but highlighted #
\``` javascript
\var x = "dummy";
\```
```

**Comment by doowb** [comment]
[See this example in the demo editor](https://jonschlinkert.github.io/remarkable/demo/#md64=eyJzb3VyY2UiOiJcbiMjIyBCYWNrdGlja3MgaW4gYmFja3RpY2tzIGV4YW1wbGVcblxuPiBVc2UgbW9yZSB0aGFuIDMgdG8gc3RhcnQgdGhlIGV4YW1wbGUsIHRoZW4gb25seSB1c2UgMyBpbnNpZGUgdGhlIGV4YW1wbGUuXG5cbmBgYGBcbnNvbWV0aGluZyBoZXJlXG5cbmBgYFxubW9yZSBzdHVmZlxuYGBgXG5cbmBgYGBcbiIsImRlZmF1bHRzIjp7Imh0bWwiOmZhbHNlLCJ4aHRtbE91dCI6ZmFsc2UsImJyZWFrcyI6ZmFsc2UsImxhbmdQcmVmaXgiOiJsYW5ndWFnZS0iLCJsaW5raWZ5Ijp0cnVlLCJ0eXBvZ3JhcGhlciI6dHJ1ZSwiX2hpZ2hsaWdodCI6dHJ1ZSwiX3N0cmljdCI6ZmFsc2UsIl92aWV3IjoiaHRtbCJ9fQ==)

Basically, use more than 3 backticks for the outer example and then 3 backticks for the inner example (this doesn't work on github)

**Comment by Outlivier** [comment]
Great, thank you :+1:

**Comment by pablobernabeu** [comment]
> [See this example in the demo editor](https://jonschlinkert.github.io/remarkable/demo/#md64=eyJzb3VyY2UiOiJcbiMjIyBCYWNrdGlja3MgaW4gYmFja3RpY2tzIGV4YW1wbGVcblxuPiBVc2UgbW9yZSB0aGFuIDMgdG8gc3RhcnQgdGhlIGV4YW1wbGUsIHRoZW4gb25seSB1c2UgMyBpbnNpZGUgdGhlIGV4YW1wbGUuXG5cbmBgYGBcbnNvbWV0aGluZyBoZXJlXG5cbmBgYFxubW9yZSBzdHVmZlxuYGBgXG5cbmBgYGBcbiIsImRlZmF1bHRzIjp7Imh0bWwiOmZhbHNlLCJ4aHRtbE91dCI6ZmFsc2UsImJyZWFrcyI6ZmFsc2UsImxhbmdQcmVmaXgiOiJsYW5ndWFnZS0iLCJsaW5raWZ5Ijp0cnVlLCJ0eXBvZ3JhcGhlciI6dHJ1ZSwiX2hpZ2hsaWdodCI6dHJ1ZSwiX3N0cmljdCI6ZmFsc2UsIl92aWV3IjoiaHRtbCJ9fQ==)
> 
> Basically, use more than 3 backticks for the outer example and then 3 backticks for the inner example (this doesn't work on github)

Working on Github too as of 2020
